sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/DocumentCollection.java

```diff
@@ -441,6 +444,35 @@ public void setVectorEmbeddingPolicy(CosmosVectorEmbeddingPolicy value) {
         this.set(Constants.Properties.VECTOR_EMBEDDING_POLICY, value, CosmosItemSerializer.DEFAULT_SERIALIZER);
     }
 
+    /**
+     * Gets the Full Text Policy containing paths for full text search and the language specification for each path.
+     * It also contains the default language to be used.
+     *
+     * @return the FullTextPolicy
+     */
+    @Beta(value = Beta.SinceVersion.V4_65_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
```

Comment:
This is implementation class, doesn't need Beta tag :)
please remove it.

Reply:
Removed it.